backend/curriculum_tracking/tests/test_UserStatsPerWeekSerializer.py

```diff
@@ -0,0 +1,219 @@
+from git_real.models import PullRequestReview
+from curriculum_tracking.serializers import UserStatsPerWeekSerializer
+from django.test import TestCase
+from curriculum_tracking.models import (
+    AgileCard,
+    ContentItem,
+)
+from . import factories
+from social_auth.tests.factories import SocialProfileFactory
+from datetime import timedelta
+from git_real.tests import factories as git_real_factories
+
+from curriculum_tracking.constants import (
+    NOT_YET_COMPETENT,
+    COMPETENT,
+)
+from curriculum_tracking.tests.factories import (
+    ContentItemFactory,
+    AgileCardFactory,
+    RecruitProjectReviewFactory,
+)
+from django.utils import timezone
+
+
+class TestingForTheStatsAPI(TestCase):
+
+    def setUp(self):
+        self.card_1 = AgileCardFactory(
+            status=AgileCard.IN_PROGRESS,
+        )
+
+        self.project_1 = self.card_1.recruit_project
+        self.user = self.card_1.assignees.first()
+        self.project_1.start_time = timezone.now() - timedelta(days=5)
+        self.project_1.save()
+
+        # Will use this repo card, PR, today, yesterday, two_days_before_yesterday and two_weeks_ago later on
+        # in test 'test_pr_reviews_card_in_different_column_due_to_review_status'.
+        self.today = timezone.now()
+        self.yesterday = self.today - timezone.timedelta(days=1)
+        self.two_days_before_yesterday = self.today - timezone.timedelta(days=3)
+        self.two_weeks_ago = self.today - timezone.timedelta(days=14)
+
+        self.repo_card_one = factories.AgileCardFactory(
+            status=AgileCard.IN_PROGRESS
+        )
+        self.repo_card_one.recruit_project.repository = git_real_factories.RepositoryFactory()
+        self.repo_card_one.save()
+
+        self.pull_request = git_real_factories.PullRequestFactory(
+            repository=self.repo_card_one.recruit_project.repository, updated_at=self.today
+        )
+        self.pull_request.save()
+
+    def test_one_card_in_progress_before_any_reviews(self):
+
+        # One card, no reviews just yet, function 'get_cards_in_progress_column' should return one card
+        assert self.card_1.status == AgileCard.IN_PROGRESS
```

Comment:
can yo uplease use `self.assertEquals`

We aren't using pytest. This unit testing framework works differently

Reply:
Made the change now.